src/__tests__/to-have-display-value.js

```diff
@@ -0,0 +1,102 @@
+import {render} from './helpers/test-utils'
+
+test('it should work as expected', () => {
+  const {queryByTestId} = render(`
+    <select id="fruits" data-testid="select">
+      <option value="">Select a fruit...</option>
+      <option value="ananas">Ananas</option>
+      <option value="banana">Banana</option>
+      <option value="avocado">Avocado</option>
+    </select>
+  `)
+
+  expect(queryByTestId('select')).toHaveDisplayValue('Select a fruit...')
+  expect(queryByTestId('select')).not.toHaveDisplayValue('Banana')
```

Comment:
Would be nice if the test also covered the case of changing the selected option programmatically to `banana` and then check that `toHaveDisplayValue('Banana')` pass.

And maybe do it for the other cases as well (textarea and input).

Reply:
Done, I'm wondering why the repository is not using `fireEvent` from the testing-library core (or even userEvent) 🤔